client/my-sites/sidebar/sidebar.jsx

```diff
@@ -343,7 +344,17 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		const planLink = '/plans' + this.siteSuffix();
+		let planLink = '/plans' + this.siteSuffix();
+
+		//Show plan details for upgraded sites
+		if (
+			config.isEnabled( 'manage/plans/my-plan' ) &&
```

Comment:
If we're placing this in an abtest, we can remove the feature `manage/plans/my-plan`.